packages/shared/components/NftMedia.svelte

```diff
@@ -38,17 +36,18 @@
 </script>
 
 {#if hasMounted && nft && nft.composedUrl && nft.parsedMetadata && (!useCaching || nft.downloadMetadata?.isLoaded)}
-    <MediaDisplay
-        {src}
-        expectedType={nft.parsedMetadata.type}
-        isLoaded={nft.downloadMetadata.isLoaded}
-        {autoplay}
-        {controls}
-        {loop}
-        {muted}
-        {classes}
-        alt={`Media display for ${nft.name}`}
-    />
+    <figure class="bg-gray-200 dark:bg-gray-700 min-w-full h-full object-cover">
```

Comment:
it might not be the best idea to make it take always the full space, because in the collectibles view, the nfts expand to all the space, which makes some ntfs look weird